packages/cli/src/lib/push/validatorErrors.ts

```diff
@@ -2,18 +2,27 @@ import {PathLike} from 'fs';
 import {CLIBaseError} from '../errors/CLIBaseError';
 
 export class NotAFileError extends CLIBaseError {
+  public name = 'Not A File Error';
   public constructor(p: PathLike) {
     super(`${p} is not a valid file, or does not exists.`);
   }
 }
+export class NotAJsonFileError extends CLIBaseError {
+  public name = 'Not A JSON File Error';
+  public constructor(p: PathLike) {
```

Comment:
I was checking what pathlike is, why not just a string really?.
`type PathLike = string | URL | Buffer` iirc. I don't think a user will input a URL object (it can't to be exact)

Reply:
I used it to stay consistent with other Validation Errors. I think the same logic would apply to other errors as well